flake.nix

```diff
@@ -146,8 +146,7 @@
 
           rustfmt = {
             enable = true;
-            # Silly test to ensure Crane's `rustfmt` check was run
-            entry = pkgs.lib.mkForce "test -d ${(mkCraneArtifacts {inherit rust;}).rustfmt}/target";
+            entry = pkgs.lib.mkForce "${rust}/bin/cargo-fmt fmt -- --check --color always";
```

Comment:
As discussed over Slack this will fail in CI.

Also as discussed, @matthew-healy will give it a try to fix by himself tomorrow, otherwise we will pair prog the fix